q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,220 @@
+# Proposal for Q2: Structure of Governance
+
+## Definitions
+
+Perl governance will consist of two groups:
+
+* "Core Team"
+
+* "Steering Council"
+
+The specific powers and composition of each are described later.
+
+A section on voting mechanics follows the description of the Core Team and
+Steering Council.  Votes are overseen by a "Vote Administrator".  There are
+five types of votes.  Two are ranked-choice:
+
+* "Term Election"
+
+* "Special Election"
+
+The other three are binary, super-majority votes:
+
+* "Membership Change Vote"
+
+* "Amendment Vote"
+
+* "No Confidence Vote".
+
+The term "Perl Rules of Governance" means the final governance document
+written and ratified by the Gov-Begin group.
+
+## The Perl Core Team
+
+The Perl Core Team are a self-managing group of trusted volunteers involved in
+the ongoing development of the Perl language and interpreter.  They are not
+required to be language developers or committers.
+
+### Powers
+
+The Core Team has authority over the rules and composition of Perl governance.
+
+They alone elect the Steering Council and have the power to remove Steering
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
+unrelated to developing and releasing the language itself.
+
+### Membership, Inactivity, and Removal
```

Comment:
PEP 13 has some useful framing copy we might want to consider borrowing.  

```
Python core team members demonstrate:

a good grasp of the philosophy of the Python Project
a solid track record of being constructive and helpful
significant contributions to the project's goals, in any form
willingness to dedicate some time to improving Python
As the project matures, contributions go beyond code. Here's an incomplete list of areas where contributions may be considered for joining the core team, in no particular order:

Working on community management and outreach
Providing support on the mailing lists and on IRC
Triaging tickets
Writing patches (code, docs, or tests)
Reviewing patches (code, docs, or tests)
Participating in design decisions
Providing expertise in a particular domain (security, i18n, etc.)
Managing the continuous integration infrastructure
Managing the servers (website, tracker, documentation, etc.)
Maintaining related projects (alternative interpreters, core infrastructure like packaging, etc.)
Creating visual designs
Core team membership acknowledges sustained and valuable efforts that align well with the philosophy and the goals of the Python project.

Core team members are always looking for promising contributors, teaching them how the project is managed, and submitting their names to the core team's vote when they're ready.
```

Reply:
I'm going to add a note that the final doc will include such a statement, but I don't want to invite wordsmithing of it now, as I understand this vote to be about structure and mechanics.

Reply:
Would you be willing to add that note to the doc?  [STATEMENT OF DUTIES WILL GO HERE IN A FUTURE ITERATION] ?  One of the confusing bits is this doc is not *just* about structure, and there are other examples of duties/responsibilities that have snuck in.  See line 42.

(FWIW, I don't think it's going to be contentious that we'll have "two branches" of government, etc.  It's the duties/responsibilities/etc. where things get messy.)

Reply:
Given feedback from you and others about being more explicit, I've added it to the revised proposal I just pushed.